### PR TITLE
feat(website): new footer and assets

### PR DESCRIPTION
**Pull request by richbachman**
Build new footer and assets.
- Top slant angle may need adjusted based on other content
- Breakpoints might need adjusted based on other content

### CODE REVIEW

**Comment by katieporter** [comment]
Looks amazing! Only thing I noticed was that the top slant starts to cut off the content on wider viewports: 
image

**Comment by serifluous** [comment]
Not sure if this meant to be addressed in this branch, but the navigation doesn't hide when viewed on mobile
Screen Shot 2020-11-12 at 9 09 14 AM

**Comment by serifluous** [comment]
Not sure how easy this'll be to clean up, but these tiny slivers of extra color in the SVG are bothering me.

cc @corinne-nagel 

Screen Shot 2020-11-12 at 9 10 55 AM

File: packages/paste-website/src/components/site-wrapper-new/SiteFooter.tsx [hunk 1]
```diff
@@ -1,35 +1,139 @@
...
+          </Box>
+          <Box
+            display="flex"
+            flexDirection={['column', 'row']}
+            marginLeft={['space0', 'space120']}
+            textAlign={['center', 'left']}
+          >
+            <Box marginRight={['space0', 'space130']} marginBottom="space60">
+              <Stack as="nav" orientation="vertical" spacing="space60">
+                <Anchor href="/about" variant="inverse">
```
Comment (serifluous) [inline packages/paste-website/src/components/site-wrapper-new/SiteFooter.tsx:83]:
```suggestion
                <Anchor href="/about-paste" variant="inverse">
```

File: packages/paste-website/src/components/site-wrapper-new/SiteFooter.tsx [hunk 2]
```diff
@@ -1,35 +1,139 @@
...
+            </Box>
+            <Box marginRight={['space0', 'space130']} marginBottom="space60">
+              <Stack as="nav" orientation="vertical" spacing="space60">
+                <Anchor href="https://github.com/twilio-labs/paste/discussions" variant="inverse" showExternal>
+                  Get support
+                </Anchor>
+                <Anchor href="https://github.com/twilio-labs/paste/issues/new/choose" variant="inverse" showExternal>
+                  Report a bug
+                </Anchor>
+                <Anchor href="/how-we-work" variant="inverse">
```
Comment (serifluous) [inline packages/paste-website/src/components/site-wrapper-new/SiteFooter.tsx:105]: Is this supposed to link to /getting-started/working-guide ?

File: packages/paste-website/src/components/site-wrapper-new/SiteFooter.tsx [hunk 3]
```diff
@@ -1,35 +1,139 @@
...
+                  How we work
+                </Anchor>
+              </Stack>
+            </Box>
+            <Box marginRight={['space0', 'space130']} marginBottom="space60">
+              <Stack as="nav" orientation="vertical" spacing="space60">
+                <Anchor href="#" variant="inverse" showExternal>
```
Comment (serifluous) [inline packages/paste-website/src/components/site-wrapper-new/SiteFooter.tsx:112]:
```suggestion
                <Anchor href="https://share.goabstract.com/786b3d8f-b87a-4140-af7d-eb4cdd40e8ed" variant="inverse" showExternal>
```

File: packages/paste-website/src/components/site-wrapper-new/SiteFooter.tsx [hunk 4]
```diff
@@ -1,35 +1,139 @@
...
+                  Abstract
+                </Anchor>
+                <Anchor href="#" variant="inverse" showExternal>
+                  Figma
+                </Anchor>
```
Comment (serifluous) [inline packages/paste-website/src/components/site-wrapper-new/SiteFooter.tsx:117]: This'll be removed before we ship, yeah? Or we could make it static text that says "Figma (coming soon)".

Comment (richbachman) [reply packages/paste-website/src/components/site-wrapper-new/SiteFooter.tsx:117]: We can remove it for now.

**Comment by richbachman** [comment]
@katieporter 

> Looks amazing! Only thing I noticed was that the top slant starts to cut off the content on wider viewports:
> image

This should be fixed now. At least as wide as I can make my browser.

**Comment by richbachman** [comment]
@serifluous 

> Not sure if this meant to be addressed in this branch, but the navigation doesn't hide when viewed on mobile
> Screen Shot 2020-11-12 at 9 09 14 AM

The breakpoints should be fixed now.

**Comment by katieporter** [comment]
> @katieporter
> 
> > Looks amazing! Only thing I noticed was that the top slant starts to cut off the content on wider viewports:
> > image
> 
> This should be fixed now. At least as wide as I can make my browser.

It still gets cloooose to cutting off the content, but it's probably okay.
image

File: packages/paste-website/src/components/site-wrapper-new/SiteFooter.tsx [hunk 5]
```diff
@@ -1,35 +1,142 @@
...
+import {Text} from '@twilio-paste/core/text';
+import HeartDoodleIcon from '../icons/HeartDoodleIcon';
+import TwilioLogo from '../icons/TwilioLogo';
+import FooterBuildingBlocks from '../../assets/illustrations/FooterBuildingBlocks.svg';
 
 const SiteFooter: React.FC<{}> = () => {
+  const theme = useTheme();
   return (
-    <StyledFooter>
+    <Box
```
Comment (SiTaggart) [inline packages/paste-website/src/components/site-wrapper-new/SiteFooter.tsx:14]: nit: I don't _love_ these large blocks of nested boxes in general. A little part of me still likes to clean stuff up like this, just like I would in the old BEM days, creating component names for different blocks, and component files.

**Review by SiTaggart — APPROVED** [review]

File: packages/paste-website/src/components/site-wrapper-new/SiteFooter.tsx (hunk 5 above)
Comment (richbachman) [reply packages/paste-website/src/components/site-wrapper-new/SiteFooter.tsx]: I broke the footer out into separate components files so they're much smaller chunks of code.

**Pull request merged**